docs/pages/enterprise/sso.mdx

```diff
@@ -32,9 +36,149 @@ organization's Single Sign-On (SSO) provider.
+
+```code
+# Log in to your cluster with tsh so you can use tctl remotely
+$ tsh login --proxy=proxy.example.com --user=myuser
+$ tctl get users
+```
 
-Execute the following command to log in to your Teleport cluster using the CLI.
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+
+```code
+# Log in to your cluster with tsh so you can use tctl remotely
+$ tsh login --proxy=mytenant.teleport.sh --user=myuser
+$ tctl get users
+```
+</ScopedBlock>
+
+Here is an example of a temporary `user` resource created when the GitHub user
+`myuser` signed in to GitHub to authenticate to Teleport. This resource
+expires 12 hours after creation. The `created_by` field indicates that the
+resource was created by Teleport's GitHub SSO integration:
+
+```yaml
+kind: user
+metadata:
+  expires: "2022-06-15T04:02:34.586688054Z"
+  id: 0000000000000000000
+  name: myuser
+spec:
+  created_by:
+    connector:
+      id: github
+      identity: myuser
+      type: github
+    time: "2022-06-14T16:02:34.586688441Z"
+    user:
+      name: system
+  expires: "0001-01-01T00:00:00Z"
+  github_identities:
+  - connector_id: github
+    username: myuser
+  roles:
+  - editor
+  - access
+  - auditor
+  status:
+    is_locked: false
+    lock_expires: "0001-01-01T00:00:00Z"
+    locked_time: "0001-01-01T00:00:00Z"
+    recovery_attempt_lock_expires: "0001-01-01T00:00:00Z"
+  traits:
+    github_teams:
+    - my-team
+    kubernetes_groups: null
+    kubernetes_users: null
+    logins:
+    - root
+version: v2
+```
+
+</Details>
+
+### Certificates for SSO users
+
+Along with creating a temporary `user` resource, Teleport issues an X.509
+certificate to a successfully authenticated SSO user. The certificate's
+`Subject` field contains the same information defined in the temporary `user`
+resource. This enables Teleport to enforce RBAC rules for the authenticated user
+when they access resources in your cluster.
+
+For example, this is a `Subject` field for a certificate that Teleport issued
+for the GitHub user `myuser`, who signed in to a Teleport cluster via the GitHub
+SSO integration:
+
+```
+Subject: L=myuser/street=teleport.example.com/postalCode={"github_teams":["my-team"],"kubernetes_groups":null,"kubernetes_users":null,"logins":["root"]}, O=access, O=editor, O=auditor, CN=myuser/1.3.9999.1.7=teleport.example.com
+```
+
+The user belongs to the GitHub team `my-team`, which this Teleport cluster maps
+to the `access`, `editor`, and `auditor` roles in Teleport. (Read the guide for
+your SSO provider to determine how to configure role mapping.)
+
+<Details title="Inspecting your certificate subject">
+
+To inspect the contents of a certificate issued for your user after you sign in to Teleport via 
+SSO, run the following commands:
+
+```code
+$ TELEPORT_CLUSTER=<your cluster>
+$ SSO_USER=<your username within your SSO provider>
+$ openssl x509 -text -in ~/.tsh/keys/${CLUSTER}/${SSO_USER}-x509.pem | grep "Subject:"
+```
+
+</Details>
+
+### Multiple SSO providers
```

Comment:
Might be worth mentioning somewhere that usernames cannot clash so if you already have a local user with the same name as your SSO username, your SSO login will fail.

Reply:
Added